projects/_posts/2018-01-01-teleirc.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: project
+title: Teleirc
+permalink: /projects/teleirc
+authors:
+- Justin Flory (@jwf)
+- Seth Hendrick (@xforever1313)
+- Mark Repka (@repkam09)
+- Nath Levesque (@thenaterhood)
+- More!
+excerpt: >
```

Comment:
I'm confused as to how the excerpt interacts with the rest of the page. This text is repeated in the About section

Reply:
It's a technical thing

Reply:
(Overrides a value that Jekyll sort of guesses at, but I wanted it to be cleaner)

Reply:
For reference, everything in the triple-dash (---) section at the top of any page in a Jekyll site is metadata Jekyll can use to do things